src/envo/envo-edit.owl

```diff
@@ -47695,6 +47701,66 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001862> "S
 ClassAssertion(<http://purl.obolibrary.org/obo/ENVO_01001211> <http://purl.obolibrary.org/obo/ENVO_01001862>)
 ObjectPropertyAssertion(<http://purl.obolibrary.org/obo/RO_0002608> <http://purl.obolibrary.org/obo/ENVO_01001862> <https://www.wikidata.org/wiki/Q525>)
 
+# Individual: <http://purl.obolibrary.org/obo/ENVO_03605201> (Albany Thicket ecoregion)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605201> "Area in the Eastern Cape of South Africa with vegetation characterized by sclerophyllous and succulent shrubs.")
```

Comment:
Best to have a more regular definition for this and other individuals. "An ecoregion in the ..."